### ISSUE TITLE
"local" loading of the app fails because of the CORS policy attributes in <script> and <link> tags

### ISSUE THREAD

**Issue by garz75** [grempe/diceware#23]
When running the diceware js application locally via the browser (file://....), the loading of the CSS and JS files fails with the following CORS policy errors (recorded on Chrome latest version, but FF also fails)

![image](https://user-images.githubusercontent.com/6645395/31689539-8e2f75b4-b344-11e7-8758-ac165e041195.png)

Removing both the integrity="sha384-...." and crossorigin="anonymous" tags on all JS and CSS fixes the issue.

**Comment by Taipo** [comment]
I assume these need to be rehashed.

**Comment by darga33** [comment]
for those of you who are trying to do what garz is saying, it works, I just did it. Just open up index.html, search for integrity, and you will see 26 matches, each something like this, but each differing: 
integrity="sha384-mXQoED/lFIuocc//nss8aJOIrz7X7XruhR6bO+sGceiSyMELoVdZkN7F0oYwcFH+" crossorigin="anonymous"

Just delete those and save the file and you are good to go! 

Thanks so much garz, how did you figure that out just by seeing that error on the CORS policy?

**Comment by SelfRando** [comment]
I can confirm that locally hosted page is completely broken in Chrome & Chromium unless these integrity tags are removed. However, it worked okay in Firefox.

**Comment by grempe** [comment]
I've fixed the improper integrity hashes (fbc65912a14324d03a530362cec3c08afa82c9c9). I am looking into the issue with Chrome throwing CORS related errors.

The proper workaround for this is not to remove the integrity hashes, but to run the local clone of the repo in a tiny webserver from the cloned directory. For example if your system has Python installed (many do by default) you should be able to do `python -m SimpleHTTPServer 8080` and then visit http://localhost:8080 in your local browser which will automatically serve the `index.html` file. Another great alternative is [https://caddyserver.com/](https://caddyserver.com/)

**Comment by grempe** [comment]
Cross origin requests are required to validate the integrity hashes. This precludes running this application from a `file:///` URL. It should therefore be run under a small local web server process as I described above. Removing the integrity checks removes an important safety factor.

I've added instructions to the README for running it this way (c5f1619edfd15a62a203620a8b90a2008aa880f0).

See:

https://stackoverflow.com/questions/41965066/access-to-image-from-origin-null-has-been-blocked-by-cors-policy
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script
https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes
